lib/kino/render.ex

```diff
@@ -80,6 +80,20 @@ end
 
 # Elixir built-ins
 
+defmodule Kino.Raw do
+  @moduledoc false
+
+  defstruct [:term]
+
+  def new(term), do: %__MODULE__{term: term}
+
+  defimpl Kino.Render do
+    def to_livebook(raw) do
+      Kino.Output.inspect(raw.term)
```

Comment:
It is probably more useful to have a `Kino.Text` that is rendered as is (and then we pass the inspected representation) rather than Kino.Raw that is inspect based?

Reply:
I don't follow, you mean calling `inspect` explicitly? That would just be more work because we would need to duplicate the inspect logic. The only reason for `Kino.Raw` is to avoid infinite loop.

Reply:
That's a good point. Here is the issue though: Axon is going to run into the same issue once they implement tabbed outputs. Should we tell them to use Kino.Raw?

Reply:
Ah totally, I made `Kino.Raw` public and we can use it in those cases.